web/src/javascript/constants.js

```diff
@@ -33,5 +33,6 @@ export const playerColors = [
 ];
 
 export const gameRefresh = 24;
+export const gameDelay = 2 * 1000;
```

Comment:
Could we add a comment to indicate what this represents? I had to spend some time looking at the code to figure it out. E.g. `// delay (ms) between the current time and the time of the currently displayed state`